_includes/figure.html

```diff
@@ -12,4 +12,7 @@
        width="{{ include.width | default: site.default-image-width }}"
        class="{{ include.class | default: "aligncenter size-full" }}"
     />
+  {% if include.description != blank %}
```

Comment:
Not sure why the `<figure>` tag has an empty `style` attribute, should be removed. figure/figcaption combo looking good otherwise; although maybe the captions should be capitalised?